routers/routes/routes.go

```diff
@@ -486,6 +486,23 @@ func RegisterRoutes(m *macaron.Macaron) {
 				m.Post("/title", repo.UpdateIssueTitle)
 				m.Post("/content", repo.UpdateIssueContent)
 				m.Post("/watch", repo.IssueWatch)
+				m.Group("/dependency", func(){
+					m.Post("/add", repo.AddDependency)
+					m.Post("/remove", repo.RemoveDependency)
+				})
+				m.Group("/times", func() {
```

Comment:
Why is this here?

Reply:
Also duplicate

Reply:
Probably because of merging with master....